src/gui/widgets/SideBarWidget.cpp

```diff
@@ -39,6 +39,10 @@ SideBarWidget::SideBarWidget( const QString & _title, const QPixmap & _icon,
 	m_layout = new QVBoxLayout( m_contents );
 	m_layout->setSpacing( 5 );
 	m_layout->setMargin( 0 );
+	m_closeBtn = new QToolButton(this);
+	m_closeBtn->setText(QString("X"));
+	connect(m_closeBtn, SIGNAL(released()),
+		this, SLOT(closeButtonClick()));
```

Comment:
You can connect a signal directly to another signal without a helper slot.

Reply:
I'm still learning Qt, and saw that it was signal->slot, but that's good to know! 😄